templates/login.template.html

```diff
@@ -39,6 +39,21 @@ <h2 class="welcomeBox__title">Stellar Account Viewer</h2>
         </div>
         <button type="button" class="s-button loginBox__submit" ng-click="login.generate()">Generate</button>
     </div>
+    <div class="middleBox">
+      Connect Ledger Nano S <img ng-src="{{login.infoImage}}" ng-mousedown="login.toggleInfo()" style="cursor: pointer;" title="Click me" />
+      <br/>
+      <div class="loginBox__info" ng-show="login.showInfo">
+        Available on Chrome and Opera. Install the Stellar app from Ledger and enable browser support in the app settings
+      </div>
+      <input type="text" ng-model="login.bip32Path" class="loginBox__input" placeholder="BIP32 path: 44'/148'/n'" />
```

Comment:
Ideally we should have a checkbox here like "Select an account" and when selected this input should be displayed. Account viewer was meant to be super simple to use and this is advanced feature.
Do you have time to implement such change? If not, I'd remove this field completely and switch to using default account.

Reply:
Yes, I can do it now. It's better to keep it as simple as possible and also to avoid confusing average user

Reply:
OK, thanks. Instead of "Select an account" it should be something like "Use default account" selected by default. When unchecked, path input should appear.

Reply:
Yes, i agree that's better.